source/releasenotes/2024-08-29-domain-verification.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Diamond and Platinum customers now exempt from domain verification enforcement"
+published_date: "2024-08-29"
+categories: [security, infrastructure]
+---
+Diamond and Platinum customers are now automatically exempt from domain verification enforcement on Pantheon when connecting custom domains in the site dashboard. Previously, an exemption had to be requested by contacting support.
+
+We are making this change to reduce the pain of adding domains for our trusted Diamond and Platinum customers.
```

Comment:
> To reduce the pain of adding domains for our trusted Diamond and Platinum customers, Diamond and Platinum customers (orgs?) are now automaticallyl exempt from domain verification enforcement. Previously, when connecting custom domains in the Pantheon site dashboard, an exemption had to be requested by contacting support. 

Better maybe? I don't like saying "Diamond and Platinum customers" twice in a row, but I think we're talking about Diamond/Platinum orgs right? So maybe the second one could be organizations? I feel like this rephrasing flows a little bit better.

Reply:
❤️ Love it. :shipit: